.github/workflows/ci-build-binaries.yml

```
@@ -80,6 +83,7 @@ jobs:
       - name: Setup versioning env vars
         run: |
           ./scripts/versions.sh | tee -a "${GITHUB_ENV}"
+      - uses: sigstore/cosign-installer@v2
```

nit: we're not consistent yet, but I'd like to make sure we're pinning these to hashes, otherwise we open up another vector for supply chain attacks

```suggestion
      - uses: sigstore/cosign-installer@9becc617647dfa20ae7b1151972e9b3a2c338a2b # v2.8.1
```

---

Done!